specs/CoreWebView2.CreateFromCOMObject.md

```diff
@@ -0,0 +1,70 @@
+CoreWebView2.CreateFromCOMObject
```

Comment:
Looking on https://learn.microsoft.com/en-us/dotnet/api/?term=comobject it looks like COM is mostly `Com` in existing .NET APIs when I search for COMObject or COMClass. Let's change COM to Com for the API name.